src/main/java/androidx/util/Base64.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.util
+
+import android.util.Base64
+import java.nio.charset.Charset
+
+/**
+ * From [String] to Base-64 encoded [String].
+ */
+fun String.toBase64(
+    offset: Int = 0,
+    len: Int = this.length,
+    flags: Int = Base64.DEFAULT
+): String = Base64.encode(toByteArray(), offset, len, flags).toString(Charset.forName("US-ASCII"))
```

Comment:
Use http://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/-charsets/-u-s_-a-s-c-i-i.html instead of looking up charset by name. And below.